src/misclibs.F

```diff
@@ -1490,25 +1413,13 @@ subroutine pdefqtest(rmax,asq,ruh,rvh,rmh,rho,q3d,dum2d1,dum2d2,dum2d3,dum2d4,du
           budt=budt+(a2-a1)
           enddo
           enddo
-          budk(k)=budt
+          asq=asq+budt*tem
         enddo
-#ifdef _B4B05R
-        !$acc update host(budk)
-#else
         !$acc end parallel
-        !$acc parallel loop vector default(present) private(k) reduction(+:asq)
-#endif
-        do k=1,nk
-          asq=asq+budk(k)*tem
-        enddo
-#ifdef _B4B05R
-        !$acc update device(asq)     
-#endif
-
       ENDIF
 
 !----------------------------------------------------------------------
-      !$acc end data
+      !!$acc end data
```

Comment:
Can we delete this line?

Reply:
@sherimickelson.  Yes, we can.  I saw this shortly after I created the PR. I have gone through and since eliminated a number of such lines.